changelog/7735.doc.rst

```diff
@@ -0,0 +1 @@
+Added a gallery example showcasing how to track and co-align an active region (:ref:`track_and_coalign_region.py`)
```

Comment:
This ref does not work, you have to use the full path including the gallery part: https://sunpy--7735.org.readthedocs.build/en/7735/whatsnew/changelog.html#dev25-g87c0ffb3a-2024-07-24